llvm/test/Transforms/LICM/funclet.ll

```diff
@@ -153,6 +153,6 @@ else:                                             ; preds = %postinvoke
 
 declare void @may_throw()
 
-declare i32 @pure_computation() nounwind argmemonly readonly willreturn
+declare i32 @pure_computation() nounwind willreturn memory(none)
```

Comment:
argmemonly for a function without arguments is memory(none). Not LICM's job to figure that out...